checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,263 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
```

Comment:
do we generally  not add Apache 2.0 copyright notice in xml files? i don't find in performance analyzer too.